### PR TITLE
fix-2624 Add OIDC overview as a linked resource

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2624
Review deadline: 6 August

## Page previews
[Azure AD SSO docs](https://deploy-preview-667--redpanda-docs-preview.netlify.app/current/manage/security/console/azure-ad/)

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [x] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/pages/security/console/azure-ad.adoc [hunk 1]
```diff
@@ -15,12 +15,12 @@ By integrating Redpanda Console with Azure AD, your users can sign in to Redpand
 You must have:
 
 * An https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/sign-up-organization[Azure AD account^] and permissions to create applications within your directory.
-* A registered OIDC application with Azure AD configured as the OpenID Connect (OIDC) provider. For instructions, see the https://learn.microsoft.com/en-us/power-apps/maker/portals/configure/configure-openid-settings[Microsoft documentation^].
+* A registered OIDC application with Azure AD configured as the OpenID Connect (OIDC) provider. For more information about using OIDC, see the https://learn.microsoft.com/en-us/entra/identity-platform/v2-protocols-oidc[Microsoft documentation^].
 +
-When you register the application, provide the following inputs when you are asked for them:
+When you https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app[register the application^], provide the following inputs when you are asked for them:
```
Comment (Feediver1) [inline modules/manage/pages/security/console/azure-ad.adoc]:
```suggestion
When you https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app[register the application^], provide the following inputs when prompted:
```

**Review by Feediver1 — APPROVED** [review]
minor wording suggestion

File: modules/manage/pages/security/console/azure-ad.adoc (hunk 1 above)
Comment (asimms41) [reply modules/manage/pages/security/console/azure-ad.adoc]: Updated

**Review by wzzzrd86 — APPROVED** [review]
Looks correct.

**Pull request merged**